user_funcs.py

```diff
@@ -116,3 +117,246 @@ def login(conn) -> tuple[str, int]:
                 return results[0]
 
             print("Username or password incorrect!")
+#THis whole function does work correctly
+def create_collection(conn, user_ID) -> None:
+    """
+    creates a new collection for the user and puts the new collection
+    in the movie collection table
+
+    modifying the collection and listing the collection are not in this method
+
+    :param conn: The connection to the database to execute SQL statements
+    :param user_ID: The ID of the user for their movie collection
+    :return: Nothing
+    """
+    #A query that will insert the movie_ID the user selected into the incollection table with associated collection_ID
+    add_movie_collection_query = """
+    INSERT INTO incollection (movieid, collectionid)
+    VALUES(%s, %s)
+    """
+
+    #Query that gets the last collection ID in the moviecollection table
```

Comment:
We can't guarantee the largest collection id is the one the user just made. What if we changed ids to be non sequential or descending? What if another user created a collection at the same time? Perhaps a more appropriate tactic would be to let this method just create a blank collection no matter what and leave movie addition for modify collection.